processes/executive-copy-creation-checklist.md

```diff
@@ -0,0 +1,57 @@
+## Creation
+
+- [ ] Sync Community Git
+- [ ] Ensure you are working from the latest master branch
+- [ ] Read Associated Executive Contents sheet
+- [ ] Copy the executive template
+- [ ] Update filename
+- [ ] Update dates - exact time is not required and should be left at midnight.
+
+- [ ] Ensure the title refers to the most impactful/important actions included in the proposal.
+- [ ] Ensure that the summary refers to all actions included in the proposal.
+
+- [ ] Ensure that the publisher of the executive is listed in the preamble (at the entity level).
+- [ ] Ensure that the writer of the executive is listed in the preamble (at the entity level).
+- [ ] Ensure that the reviewers of the executive are listed in the preamble (at the entity level).
+
+- [ ] Ensure that the office-hours modifier is mentioned as part of the executive copy, if present in the executive spell code.
+- [ ] Ensure that it is clearly stated which items are affected by the GSM Delay.
+- [ ] Ensure that the GSM Delay period is clearly communicated.
+- [ ] Ensure that the proposal expiry is explicitly noted, including the length of time for which the proposal is valid.
+
+## Items
+
+- [ ] Add each item to the proposal details section
+- [ ] Order the items by judged impact/importance
+
+- [ ] For each item
+	- [ ] Ensure the executive summary contains a high-level overview of every action contained in the proposal.
+	- [ ] Ensure that each item or set of items in the executive proposal has a corresponding entry in the proposal details section.
+	- [ ] Ensure that each item in the copy contains a link to the reasoning for its inclusion.
+	- [ ] Ensure that each item in the copy contains a link to the authorization for its inclusion.
+	- [ ] Ensure that each item in the copy contains any links required for voters to determine the context surrounding its inclusion.
+	- [ ] Ensure that the framing and tense used for each item is future-conditional. Ie: "IF this executive proposal passes, THEN this action will take place."
+	- [ ] Ensure that hyperlinks to documentation and/or context are added to key parameters, terms, and contracts where reasonable and possible.
+	- [ ] Ensure that Ethereum and IPFS addresses are hyperlinked such that it is easy to view their details.
+	- [ ] Ensure that parties implicated in the proposal outcome related to this item are aware of the wording used.
+
+## Self-Review
+
+- [ ] Do any '$' parameter signifiers remain? (spell address is unknown at this point)
+- [ ] Do the executive title, summary, and executive summary accurately reflect executive copy content?
+- [ ] Read the executive copy in full, with the above checklists and general readability in mind. Make any required changes.
+- [ ] Run the copy through a spelling and grammar checker - do not blindly accept suggested changes.
+- [ ] Inform the reviewer that the executive copy is ready to review
+- [ ] Commit changes, and create a PR to the Community GitHub repository.
+
+## Submission
```

Comment:
add provide copy to growth team for twitter?

Reply:
I could do, this checklist basically already exists in the revamped executive sheet. That includes all the external wants. I'm tempted to either remove this checklist from here, or just make it clear that these are the 'internal' requirements, and there are other external things to do.